packages/web-components/src/components/pricing-table/pricing-table-cell-annotation.ts

```diff
@@ -9,12 +9,13 @@
 
 import { customElement, LitElement, property, html } from 'lit-element';
 import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings';
+import StableSelectorMixin from '../../globals/mixins/stable-selector';
 import styles from './pricing-table.scss';
 
 const { stablePrefix: ddsPrefix } = ddsSettings;
 
 @customElement(`${ddsPrefix}-pricing-table-cell-annotation`)
-class DDSPricingTableCellAnnotation extends LitElement {
+class DDSPricingTableCellAnnotation extends StableSelectorMixin(LitElement) {
```

Comment:
There will be a merge conflict with the changes in this PR (https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/8993/files#diff-2b05748087218281bf3f37742976a0308c629db96f8ac0e9a745bc60483cc55cR18).